template_wizard.Makefile

```diff
@@ -0,0 +1,111 @@
+.PHONY: template_wizard_clean template_wizard_all 
+
+template_wizard_all: template_wizard_clean target/output/schema.yaml
+
+template_wizard_clean:
+	rm -rf target/output/*tsv
+	rm -rf target/output/*csv
+	rm -rf target/output/*.yaml
+	rm -rf target/templates/generated*
+	rm -rf target/output/generated*
+	mkdir -p target/input
+	mkdir -p target/templates
+	mkdir -p target/output
+
+
+#target/output/slot.tsv:
+#	poetry run linkml2sheets target/templates/*.tsv \
+#  		--schema /Users/MAM/Documents/gitrepos/linkml-model/linkml_model/model/schema/meta.yaml  \
+#  		--output-directory target/output \
+#  		--overwrite
+  		
+# target/output/schema.yaml: target/output/slot.tsv
+#	poetry run sheets2linkml target/output/*.tsv > $@
+
+target/output/key_counts.csv:
+	poetry run python schemasheets/list_element_slots.py
+
+# template_wizard_clean
+
+target/output/NMDC_schema_slot_class_merged_with_filtered_rels.tsv:
```

Comment:
this doesn't look like something that belongs in a general purpose schemasheets repo?

Reply:
Happy to submit variables to replace the specific if desired.